### ISSUE TITLE
Special characters inside <td> are not handled correctly

### ISSUE THREAD

**Issue by RudiSchloesser** [zopefoundation/z3c.rml#92]
The & character is not handled correctly within a <td>:
I tried 
  1. <td>Some text &amp; some more text</td>
  2. <td><![CDATA[Some text &amp; some more text]]></td>

1. gives &amp; in the PDF output.
2. gives &amp;amp; in the PDF output

**Comment by jonnya** [comment]
I have been having a similar issue, but working some of the time.
`<td>Some text &amp; some more text</td>` would only produce &amp; in the pdf output.
On closer inspection of my input the ones that worked were wrapped in a <para> tag

`<td><para>Some text &amp; some more text</para></td>`  produced & in the pdf output.
Don't know if this helps, but reading your posts led me to find a solution for my issue.

**Comment by regebro** [comment]
Yes, to have text inside a cell behave like text it should be wrapped in a para tag.
